_components/site-alert/guidance/accessibility.md

```diff
@@ -1 +1,37 @@
-- **Don’t visually hide alert messages and then make them visible when they are needed.** Users of older assistive technologies may still be able to perceive hidden alert messages. Fully remove alert messages when they’re not needed.
\ No newline at end of file
+- **Don’t visually hide alert messages and then make them visible when they are needed.** Users of older assistive technologies may still be able to perceive hidden alert messages. Fully remove alert messages when they’re not needed.
+- **Include a descriptive ARIA label.** Add an `aria-label` or `aria-labelledby` attribute to the `.usa-site-alert` element to ensure that your site alert appears in the landmarks menu. A well-written label can help assistive technology users easily find and understand the top-level information of your site alert.
+- **Add an ARIA role to dynamically-updated site alerts.** If your site alert will be dynamically updated during a session, add an ARIA `role` attribute to notify assistive technologies users of important and time-sensitive changes. To elevate importance, add the appropriate `role` to the `.usa-site-alert` element, chosen from the following table:
+
```

Comment:
Add a title to the table. It's best practice (I'm learning) that tables should have titles so that they can be referred to in the instructions while avoiding directional instructions like: "below, above, following."

Reply:
Thanks for the suggestion. Added in [c7f0d24](https://github.com/uswds/uswds-site/pull/1774/commits/c7f0d249cd77e2862fd5ad133a1b59804c643f6f)